hedera-node/src/main/java/com/hedera/services/legacy/crypto/SignatureStatusCode.java

```diff
@@ -47,7 +47,10 @@
   UNRESOLVABLE_REQUIRED_SIGNERS(
           "Cannot resolve required signers for scheduled txn [ source = '%s', scheduled = '%s', error = '%s' ]"),
   SCHEDULED_TRANSACTION_NOT_IN_WHITELIST(
-          "Specified txn cannot be scheduled [ source = '%s', transactionId = '%s' ]");
+          "Specified txn cannot be scheduled [ source = '%s', transactionId = '%s' ]"),
+  INVALID_FEE_COLLECTOR(
+          "Fee collector for custom account does not exist [ source = '%s', transactionId = '%s' ]");
```

Comment:
`custom account` ---> `Custom Fee Schedule` ?

Reply:
`Fee collector account for a custom fee does not exist...`
Perhaps we should change the code into `INVALID_CUSTOM_FEE_COLLECTOR`

Reply:
`SignatureStatusCode` is essentially dead code (it could be removed completely and only `ResponseCodeEnum`s used), but will require some work..